mycroft/skills/core.py

```diff
@@ -1648,6 +1650,26 @@ def cancel_all_repeating_events(self):
         for e in list(self.scheduled_repeats):
             self.cancel_scheduled_event(e)
 
+    def acknowledge(self):
+        """ Acknowledge a successful request.
+
+        This method plays a sound to acknowledge a request that does not
+        require a verbal response. This is intended to provide simple feedback
+        to the user that their request was handled successfully.
+        """
+        file = resolve_resource_file(
+            self.config_core.get('sounds').get('acknowledge'))
+
+        if not file:
+            LOG.warning("Could not find 'acknowledge' audio file!")
+            return
+
+        process = play_audio_file(file)
+        if process:
+            process.wait()
```

Comment:
Should this be blocking by default? maybe a parameter `wait` defaulted to False?

Reply:
I was referencing https://github.com/MycroftAI/mycroft-core/blob/dev/mycroft/client/speech/mic.py#L517 which uses `wait`. I suppose there it's more necessary because we want to wait for the sound to play before we start listening to the user (I would assume we run the risk of the sound being recorded otherwise, and impeding the STT). Here it's probably less necessary/less useful, since it's a separate process and there's nothing to stop the wake word/STT/other processes anyway. I'll remove it.